src/power_grid_model/_core/power_grid_meta.py

```diff
@@ -205,3 +205,40 @@ def initialize_array(
         dtype=power_grid_meta_data[data_type][component_type].dtype,
         order="C",
     )
+
+
+def attribute_dtype(
+    data_type: DatasetTypeLike, component_type: ComponentTypeLike, attribute: AttributeType
+) -> np.dtype:
+    """Gives out dtype of the attribute to be used in a columnar data format
+
+    Args:
+        data_type (DatasetTypeLike): The type of dataset (input, update, sym_output, or asym_output)
+        component_type (ComponentTypeLike): The type of component (e.g., node)
+        attribute (AttributeType): The attribute whose dtype is required
+
+    Returns:
+        np.dtype: The dtype of the specified attribute
+    """
+    data_type = _str_to_datatype(data_type)
+    component_type = _str_to_component_type(component_type)
+    return power_grid_meta_data[data_type][component_type].dtype[attribute]
+
+
+def attribute_empty_value(
```

Comment:
Maybe consider naming this one to something more expressive? Like `set_empty_attribute` , `set_nan_attribute`, or something along those lines?

Reply:
You are right, then the naming makes perfect sense. Resolving.